src/main/java/mate/academy/Main.java

```diff
@@ -1,16 +1,50 @@
 package mate.academy;
 
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import mate.academy.lib.Injector;
+import mate.academy.model.CinemaHall;
 import mate.academy.model.Movie;
+import mate.academy.model.MovieSession;
+import mate.academy.service.CinemaHallService;
 import mate.academy.service.MovieService;
+import mate.academy.service.MovieSessionService;
 
 public class Main {
-    public static void main(String[] args) {
-        MovieService movieService = null;
+    private static final Injector injector = Injector.getInstance("mate.academy");
 
+    public static void main(String[] args) {
+        final MovieService movieService =
+                (MovieService) injector.getInstance(MovieService.class);
         Movie fastAndFurious = new Movie("Fast and Furious");
         fastAndFurious.setDescription("An action film about street racing, heists, and spies.");
         movieService.add(fastAndFurious);
-        System.out.println(movieService.get(fastAndFurious.getId()));
+        System.out.println("Movie added: " + movieService.get(fastAndFurious.getId()));
+        System.out.println("All movies:");
         movieService.getAll().forEach(System.out::println);
+
+        final CinemaHallService cinemaHallService =
+                (CinemaHallService) injector.getInstance(CinemaHallService.class);
+        CinemaHall bigHall = new CinemaHall();
+        bigHall.setCapacity(200);
+        bigHall.setDescription("Big cinema hall with surround sound.");
+        cinemaHallService.add(bigHall);
+        System.out.println("CinemaHall added: " + cinemaHallService.get(bigHall.getId()));
```

Comment:
The `get()` method of `cinemaHallService` should return an `Optional<CinemaHall>` instead of `CinemaHall`. This change is necessary to handle potential null values safely, according to the task requirements.